Assignment_5.R

```diff
@@ -0,0 +1,185 @@
+# volume in space (the ball of radius 11 megaparsecs centered on you and me). 
+# There are, however, some signs that the smaller objects are under-represented in the sample.
+# Can you make a plot that reveals this tendency and a likely explanation?
+#
+## Load necessary packages -------
+library(ggplot2)
+
+
+
+
+
+#
+## Histogram and scatterplot-------
+histogram <- ggplot(galaxies, aes(x = a_26)) + 
+  geom_histogram(binwidth = 0.8, fill = "red", alpha = 0.9) + 
+  labs(title = "Distribution of Galaxies (by size)", 
+       x = "Size", 
+       y = "Number of galaxies") +
+  scale_x_continuous(breaks = seq(0, 70, by = 5), minor_breaks = seq(0, 70, by = 1)) +
+  scale_y_continuous(breaks = seq(0, 170, by = 25)) +
+  theme_minimal() +
+  theme(
+    plot.title = element_text(hjust = 0.5),
+    panel.grid.major = element_line(color = "grey50", linetype = 1, size = 0.2),
+    panel.grid.minor = element_line(color = "grey", linetype = "dotted", size = 0.5)
+  )
+histogram
+
+
+scatterplot_distance <- ggplot(galaxies, aes(x = a_26, y = D, color = m_b)) + 
+  geom_point(alpha = 0.5) +
+  geom_smooth(formula = y ~ x, method = loess) +
+  guides(color = guide_legend(override.aes = list(linetype = 1)), fill = TRUE) +
+  labs(title = "Scatterplot of Size and Distance", 
+       x = "Size (kpc)", 
+       y = "Distance (Mpc)") +
+  scale_x_continuous(breaks = seq(0, 60, by = 5)) +
+  theme_minimal() +
+  theme(
+    plot.title = element_text(hjust = 0.5),
+    panel.grid.minor = element_line(color = "grey", linetype = "dotted", size = 0.5)
+  )
+
+scatterplot_distance
+
+### Explanation ------
+# Histogram display the distribution of galaxies by size, in which the majority of galaxies are distributed at
+# the lower range of sizes. Somewhat surprisingly, there are relatively few of the smallest galaxies in the first bin,
+# compared to the subsequent 6 bins. From looking at the histogram alone, it could be inferred that the smallest
+# galaxies of size ~ 1 kpc are significantly less numerous than galaxies of size ~2-5 kpc. Galaxies larger than 5 kpc
+# are decreasing in size, e.g., increasingly rare. However, it cannot be inferred from the histogram alone that
+# smaller objects are under-represented in the sample, and not displaying the natural distribution of galaxies´ size.
+# 
+# Looking at the scatterplot, the relationship between size and distance to galaxy is displayed. There are
+# no obvious signs that our detection methods are less effective for smaller galaxies that are farther away.
+# What we do see is that the smoother line using the default LOESS method to fit a smooth curve does not include the smallest galaxies,
+# which could indicate that smaller galaxies might be underrepresented in the dataset.
+# However, it might also indicating a bias in the dataset towards larger galaxies. Using the lm-method fixes this - displaying the
+# line using linear regression method.
```

Comment:
You could have plotted a scatter plot with mass vs distance. The plot would reveal a tendency that small galaxies are observed more frequently close to us than far away from us. This may indicate that small galaxies are hard to notice when they are further away.

Reply:
I actually did just this in my draft but excluded it as i was unsure whether my reasoning was correct...